### PR TITLE
Less v2

### PR DESCRIPTION
**Pull request by i-like-robots**

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -1,10 +1,7 @@
 web: forever server/app.js
 v1Api: forever server/update-search.js --index v1_api_v2 --type item
-v2Api: forever server/update-search.js --index v2_api_v2 --type item --apiVersion 2
```
Comment (matthew-andrews) [inline Procfile]: we still need this one i think because for notifications feed this triggers V3 ingests :)

File: README.md [hunk 2]
```diff
@@ -23,7 +23,6 @@ Currently only supports V1 content.
 `update-search.js` keeps the search indices up-to-date. It is executed as follows:
 
     v1Api: forever server/update-search.js --index v1_api_v2 --type item
-    v2Api: forever server/update-search.js --index v2_api_v2 --type item --apiVersion 2
```
Comment (matthew-andrews) [inline README.md]: i think this one is still needed

File: Procfile (hunk 1 above)
Comment (i-like-robots) [inline Procfile]: Maybe we should rename it =]

Comment (matthew-andrews) [inline Procfile]: good idea :+1:

**Comment by matthew-andrews** [comment]
i'm going to break thsi into smaller pieces

**Pull request closed without merging**